ios/IoReactNativeSecureStorage.swift

```diff
@@ -1,8 +1,142 @@
 @objc(IoReactNativeSecureStorage)
-class IoReactNativeSecureStorage: NSObject {
 
-  @objc(multiply:withB:withResolver:withRejecter:)
-  func multiply(a: Float, b: Float, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
-    resolve(a*b)
+/// React Native bridge which leverages ``SecureStorage`` functionality to store content securly via Keychain.
+/// Methods are run in a custom serial queue to syncronize access to the Keychain, thus only one block can run at any time to
+/// syncronize the Keychain access.
+/// Each string passed to the `put` and `get` is encoded in UTF-8.
+class IoReactNativeSecureStorage: NSObject {
+  
+  private typealias ME = ModuleException
+  
+  private let storage = SecureStorage(serviceName: "rn-secure-storage");
+  
+  let queue = DispatchQueue(label: "rn-secure-storage-queue")
+  
+  /// See ``SecureStorage/put(key:data:)``
+  /// `data` is encoded in UTF-8.
+  @objc(put:withData:withResolver:withRejecter:)
+  func put(key: String, data: String, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) -> Void {
```

Comment:
To get same code style and to improve function signature readability used in others repository I suggeste to adopt this template. (see also https://github.com/pagopa/io-react-native-integrity/blob/0773aeeb4f3f071241eb4d8a1a608e60b0ce9601/ios/IoReactNativeIntegrity.swift#L13-L19) 

```suggestion
  @objc(put:withData:withResolver:withRejecter:)
  func put(
    key: String, data: String, 
    resolve: @escaping RCTPromiseResolveBlock, 
    reject: @escaping RCTPromiseRejectBlock
  ) -> Void {
```

Reply:
Is there any way we can enforce it via Xcode? Thanks for the suggestion.

Reply:
There is a nicely short cut. `control^+M` and the line where the cursor is on will be re-indent on multiple lines.

Reply:
Done in 24daeb20da4836597682484eee7acd3419eed64f, thanks for the suggestion.